notifications/src/main/java/com/dojo/notifications/model/SlackMethodsLookupByEmail.java

```diff
@@ -0,0 +1,46 @@
+package com.dojo.notifications.model;
+
+import com.hubspot.slack.client.methods.JsonStatus;
+import com.hubspot.slack.client.methods.MethodWriteMode;
+import com.hubspot.slack.client.methods.RateLimitingTier;
+import com.hubspot.slack.client.methods.RateLimitingTiers;
+import com.hubspot.slack.client.methods.SlackMethod;
+
+public enum SlackMethodsLookupByEmail implements SlackMethod {
+    users_lookupByEmail(MethodWriteMode.READ, RateLimitingTiers.TIER_3, JsonStatus.ACCEPTS_JSON)
+    ;
+
+    private final MethodWriteMode writeMode;
+    private final RateLimitingTier rateLimitingTier;
+    private final JsonStatus jsonStatus;
+
+    SlackMethodsLookupByEmail(
```

Comment:
This is not too large declaration to make parameters on separate lines - it is harder to read this way